frontend/src/components/VideoCardPage/VideoCardPage.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import Grid from '@material-ui/core/Grid';
+import TopBar from '../TopBar/TopBar';
+import UploadModal from '../UploadModal/UploadModal';
+import './styles.css';
+import VideoCardsByDate from '../VideoCardsByDate/VideoCardsByDate';
+import NavDrawer from '../NavDrawer/NavDrawer';
+import CustomSnackbar from '../CustomSnackbar/CustomSnackbar';
+import DeleteConfirmationDialog from '../DeleteConfirmationDialog/DeleteConfirmationDialog';
+
+const VideoCardPage = ({
+  renderSnackbars,
+  title,
+  size,
+  showNavDrawer,
+  videosInformation,
+  showUploadModal,
+  showDeletionDialog,
+  selectedCardIds,
+  showDeletionError,
+  iconsToShow,
+  iconsToShowOnSelected,
```

Comment:
Not sure I like that many props being passed down, I think it's a bit overkill. But you already did it, so I don't think you should be reversing it back